service.rsspopup/service.py

```diff
@@ -0,0 +1,212 @@
+
+import time, sys, os, urlparse,json
+import xbmc ,xbmcgui, xbmcaddon,xbmcvfs
+import urllib2,urllib,json
+import shutil
+import re,md5
+import socket, cookielib
+import feedparser
+import popupwindow
+import HTMLParser
+from dateutil import parser
+
+__addon__ = xbmcaddon.Addon()
+__addonname__ = __addon__.getAddonInfo('name')
+__addondir__    = xbmc.translatePath( __addon__.getAddonInfo('path') )
+background = os.path.join(__addondir__,"bg.png")
+
+profile    = xbmc.translatePath( __addon__.getAddonInfo('profile') ).decode("utf-8")
+temp       = xbmc.translatePath( os.path.join( profile, 'temp', '') ).decode("utf-8")
+translation = __addon__.getLocalizedString
+
+
+  
+wid = xbmcgui.getCurrentWindowId()
+window=xbmcgui.Window(wid)
+window.show()
+        
+def debug(content):
+    log(content, xbmc.LOGDEBUG)
+    
+def notice(content):
+    log(content, xbmc.LOGNOTICE)
+
+def log(msg, level=xbmc.LOGNOTICE):
+    addon = xbmcaddon.Addon()
+    addonID = addon.getAddonInfo('id')
+    xbmc.log('%s: %s' % (addonID, msg), level) 
+    
+    
+# Einlesen von Parametern, Notwendig fuer Reset der Twitter API
+def parameters_string_to_dict(parameters):
+	paramDict = {}
+	if parameters:
+		paramPairs = parameters[1:].split("&")
+		for paramsPair in paramPairs:
+			paramSplits = paramsPair.split('=')
+			if (len(paramSplits)) == 2:
+				paramDict[paramSplits[0]] = paramSplits[1]
+	return paramDict
+
+if not xbmcvfs.exists(temp):
+   xbmcvfs.mkdirs(temp)
+
+    
+def addrss():
+    dialog = xbmcgui.Dialog()
+    d = dialog.input("Url des Feeds", type=xbmcgui.INPUT_ALPHANUM)
+    filename       = xbmc.translatePath( os.path.join( temp, 'urlliste.txt') ).decode("utf-8")
+    if xbmcvfs.exists(filename) :
+      fp=open(filename,"r") 
+      content=fp.read()
+      fp.close()    
+      content=content+"\n"+d
+    else :
+       content=d
+    fp = open(filename, 'w')    
+    fp.write(content)
+    fp.close()    
+      
+def deleterss():
+   filename       = xbmc.translatePath( os.path.join( temp, 'urlliste.txt') ).decode("utf-8")
+   if xbmcvfs.exists(filename) :
+      fp=open(filename,"r") 
+      content=fp.read()
+      fp.close()          
+      liste=content.split("\n")
+      dialog = xbmcgui.Dialog()
+      nr=dialog.select(translation(30001), liste)
+      if nr<0:
+        return
+      liste.remove(liste[nr])
+      content="\n"
+      content=content.join(liste)
+      fp = open(filename, 'w')    
+      fp.write(content)
+      fp.close()    
+      
+   
+   
+# Soll Twitter Api Resetter Werden
+if len(sys.argv) > 1:
+    params = parameters_string_to_dict(sys.argv[2])
+    mode = urllib.unquote_plus(params.get('mode', ''))
+    if mode=="addrss":
+      addrss()
+    if mode=="deleterss":
+      deleterss()
+    
+
+
+
+def geturl(url):
+   req = urllib2.Request(url)
+   inhalt = urllib2.urlopen(req).read()   
+   return inhalt    
+          
+    
+if __name__ == '__main__':
+    cimg=""
+    xbmc.log("Twitter:  Starte Plugin")
```

Comment:
YOu didn't understand. Instead of calling xbmc.log just use the debug method you already have on the addon. so: debug("Plugin started"). The messages must be in english, since the user might be interested in checking the log file as well. Reading messages in german for non-german users is not something I would like to have on my logs